source/helpers/engines.ts

```diff
@@ -7,12 +7,25 @@ import path from 'path';
 
 import { renderPermalink } from './components/anchors';
 
+/**
+ * Identical to `markdown-it-anchor`'s default slugify function, but removes
+ * leading dashes to match the behavior of the old Ruby site.
```

Comment:
In the current site, a section with a heding like[ `<h3>--quiet</h3>`](https://sass-lang.com/documentation/cli/dart-sass#quiet) would produce an anchor link to `#quiet` (leading dashes omitted). In the new site, `markdown-it-anchor` was keeping the dashes so the anchors were being generated as `#--quiet`, which breaks old links. This change preserves the old behavior.

Reply:
I resolved this similarly in #50 by using https://github.com/sindresorhus/slugify.